torch_xla_py/xla_env_vars.py

```diff
@@ -0,0 +1,8 @@
+MP_DEVICE = 'XRT_MULTI_PROCESSING_DEVICE'
+LOCAL_WORKER = 'XRT_LOCAL_WORKER'
+TPU_CONFIG = 'XRT_TPU_CONFIG'
+SERVICE_ADDRESS = 'XRT_MESH_SERVICE_ADDRESS'
+DEVICE_MAP = 'XRT_DEVICE_MAP'
+WORKERS = 'XRT_WORKERS'
+ORDINAL = 'XRT_SHARD_ORDINAL'
```

Comment:
If I'm understanding this PR correctly, this should work just fine with `xla_dist.py`?

`XRT_LOCAL_WORKER` is still per VM, `XRT_SHARD_ORDINAL` is overridden (so no need to set this anymore in `xla_dist.py` unless we're not using MP for distributed training), `WORLD_SIZE` is still the total number of VMs.

Reply:
There should not be any changes in `xla_dist.py`. That module should not know anything about multi-threading vs. multi-processing.
We should continue to export what we do, as some information are used by our new multiprocessing, to properly setup the environment.

Reply:
shall we stick to pytorch lingo? I think they call this "RANK"

Reply:
I have no strong opinion, but to me "ordinal" sounds better.
But isn't RANK the world size?
Anyway, this involves changes in xla dist as well, so might be topic for a different PR IMHO.

Reply:
IIRC RANK is the index of a particular machine, world size is total num of devices (or processes, not sure). I'm fine either way, ORDINAL it is.